tests/database/mysql/test_query_factory.py

```diff
@@ -61,6 +61,12 @@ def int_fake_column_generator():
         get_data_type=Mock(return_value=FakeDataType.INT),
         get_value=Mock(return_value=645)
     )
+@pytest.fixture
+def uuid_fake_column_generator():
+    return Mock(
+        get_data_type=Mock(return_value="UUID"),
```

Comment:
I think there might be a misunderstanding here about the use of this fixture. 
it's mocking a `FakeColumnGenerator`. get_data_type should return something from the FakeDataType enum - actually a FakeDataType.STRING in this case. 

https://github.com/jerometwell/pynonymizer/blob/master/pynonymizer/fake/__init__.py#L56

I know that they are string values eventually, but I don't think this is a good idea for the tests.